app/src/main/assets/config.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Should not be loaded from assets.

The POJO should have default values and loaded from a file in internal memory, if it does not exist, the default POJO will be returned.

And we will change the settings and save the modified file there only

Reply:
@iamareebjamal Correct me if I am wrong. You mean to say 

- I should put my config.json in the internal memory and not assets folder.
- POJO should have default values (for those cases where the file in the internal memory doesn't exist). In this case the defalt values would be used to populate checkboxes and the same default values would be used to create a json file in the internal memory(because it does not exist till now)
- For the cases where the file exists in the internal memory simply load them into POJO and use the corresponding values.

Reply:
There will be no config.json shipped with the app. Only generated when settings from app change.

And yes, you understand correctly